cmd/ferretdb/main.go

```diff
@@ -40,6 +44,7 @@ var (
 	proxyAddrF       = flag.String("proxy-addr", "127.0.0.1:37017", "")
 	versionF         = flag.Bool("version", false, "print version to stdout (full version, commit, branch, dirty flag) and exit")
 	testConnTimeoutF = flag.Duration("test-conn-timeout", 0, "test: set connection timeout")
+	handlerF         = flag.String("handler", "pg", "handler: can be pg or dummy")
```

Comment:
I think we should provide information about the flag's purpose and then provide available options like `handler: set backend handler (pg, dummy)`

Reply:
fixed